### ISSUE TITLE
dependency updates in 13.0.8 make bingads-api unusable for me

### ISSUE THREAD

**Issue by schabe77** [BingAds/BingAds-Java-SDK#123]
I'm using openjdk 11 and have `com.google.api-ads:adwords-axis` as a dependency in my project.  I depend on bingAds and googleAds to work together. Up to now it's was not easy to make all these concurrent version work together.
I had to exclude `jakarta.mail:jakarta.mail-api`, because it depends on `com.sun.mail.util.PropUtil` that doesn't exist in openjdk and I had to exclude `com.monitorjbl:xlsx-streamer` because it's xerces implementation collides with axis' requirements.

The problems I have now are:
1. updating CXF dependency version from 3.3.2 to 3.4.2. removed `javax.xml.ws:jaxws-api` from dependencies (these classes are used by `com.microsoft.bingads.ServiceClient` amongst others). No big deal, the dependency can be added manually

2. updating jakarta.jws-api version from 1.1.1 to 3.0.0. This is a big problem. They changed the package from `javax.jws` to `jakarta.jws`, so `javax.jws.WebService` used in `com.microsoft.bingads.internal.ServiceFactoryImpl` doesn't exist anymore. Besides: this package is used by adwords-axis too in version 1.1.1 - this update not only breaks bingAds but googleAds too. Even if I provide the original package-format using another artifact (`javax.jws:javax.jws-api`), I still get the exception
```
Caused by: javax.xml.soap.SOAPException: Unable to create message factory for SOAP: Unable to create SAAJ meta-factory: Provider com.sun.xml.internal.messaging.saaj.soap.SAAJMetaFactoryImpl not found
        at javax.xml.soap.MessageFactory.newInstance(MessageFactory.java:90)
        at org.apache.cxf.binding.soap.saaj.SAAJFactoryResolver.createMessageFactory(SAAJFactoryResolver.java:56)
        at org.apache.cxf.binding.soap.saaj.SAAJOutInterceptor.getFactory(SAAJOutInterceptor.java:87)
        at org.apache.cxf.binding.soap.saaj.SAAJOutInterceptor.handleMessage(SAAJOutInterceptor.java:123)
```
and I don't know how to create a workaround for this issue.

Long story short: I don't know if it's possible to use 13.0.8 at all, at least for openjdk I'm not able to use it.

**Comment by qitia** [comment]
thanks for reporting. we will re-evaluation this version.

**Comment by qitia** [comment]
@schabe77 We published version 13.0.8.1 for compatible with java 11. please try this version and let me know for any found. Thanks.

**Comment by prisams** [comment]
This works.
